nubis/bin/example-consul2unixusers.conf

```diff
@@ -0,0 +1,5 @@
+[consul]
+server = hostname.for.your.consul.server
+scheme = http
+port = 80
```

Comment:
Wouldn't making it a consul url make more sense:
[consul]
url = http://hostname.for.your.consul.server:80/

And really, should also have defaults to http://localhost:8500/